src/pages/ResetPassword/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Container, FormContainer } from './style';
+import { useLocation } from 'react-router-dom';
+import { api, history } from '../../services';
+import { toast } from 'react-toastify';
+
+function useQuery() {
+    return new URLSearchParams(useLocation().search);
+}
+
+const ResetPassword = () => {
+    const query = useQuery();
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [resetErrors, setResetErrors] = useState({});
+
+    async function postResetPassword(e) {
+        e.preventDefault();
+
+        try {
+            await api.post('/api/password/reset', {
+                email: email,
+                password: password,
+                token: query.get('token'),
+            });
+            history.push('/');
+        } catch (error) {
+            const { message, errors } = error.response.data;
+            setResetErrors({
+                ...resetErrors,
+                ...errors,
+            });
```

Comment:
Na maneira como esta você não reseta os erros a cada submit, só ajunta.
Uma sugestão.
```suggestion
            setResetErrors(errors);
```

Reply:
@lucasemanuel8 essa rota precisa de autenticação? como você tá verificando ali na sugestão. 👆🏾👆🏾👆🏾